pkg/tracker/index.go

```diff
@@ -100,7 +99,7 @@ func parseIndexFile(logger log.Logger, cfg *config.Config, DB db.DataServerProxy
 					}
 				case fileIndexType:
 					{
-						source = &JSONfile{filepath: filepath.Join(cfg.ConfigFolder, api.URL)}
+						source = &JSONfile{filepath: filepath.Join("configs", api.URL)}
```

Comment:
We should not have hard codings for paths in the code! this is very important!
In this case this needs to be set in the api.json file as:
"URL": "configs/manualData.json",